lib/features/session/domain/usecases/get_session_interactor.dart

```diff
@@ -11,9 +13,35 @@ class GetSessionInteractor {
 
   Stream<Either<Failure, Success>> execute() async* {
     try {
+      yield Right<Failure, Success>(GetSessionLoading());
+      if (PlatformInfo.isMobile) {
+        yield* _getStoredSession();
```

Comment:
ahh, only mobile support `offlineMode`. OK.
But I think the first logic is better, if can not get session from server, please get session from store, because session maybe change time to time, when somebody have new `limit`, `quota`, bla bla